configuration/openmrs/obscalculator/BahmniObsValueCalculator.groovy

```diff
@@ -55,26 +58,47 @@ public class BahmniObsValueCalculator implements ObsValueCalculator {
         Collection<BahmniObservation> observations = bahmniEncounterTransaction.getObservations()
         def nowAsOfEncounter = bahmniEncounterTransaction.getEncounterDateTime() != null ? bahmniEncounterTransaction.getEncounterDateTime() : new Date();
 
-        BahmniObservation heightObservation = find("Height", observations, null)
-        BahmniObservation weightObservation = find("Weight", observations, null)
+ BahmniObservation heightObservation = findByUUID("5090AAAAAAAAAAAAAAAAAAAAAAAAAAAA", observations, null);
```

Comment:
Could you specify this as a constant at the top of the file? and substitute any further references by the constant.